src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -129,7 +130,8 @@ public void start(final Map<String, String> parsedConfig) {
       // connector would never start or reach the sink task stage
       behavior =
           SnowflakeSinkConnectorConfig.BehaviorOnNullValues.valueOf(
-              parsedConfig.get(SnowflakeSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG));
+              parsedConfig.get(SnowflakeSinkConnectorConfig.BEHAVIOR_ON_NULL_VALUES_CONFIG)
```

Comment:
Is this safe when no option is specified, does this get filled in with the default or is it `null`?

Reply:
It checks whether an option is provided here. If there is no option it uses default on line 127. And if an option is specified, it should have been validated by the validator, so we can assume it's always valid here.